Adding Russian language for quest books + common language

clarkins1 opened the pull request:
## What
<!--This section describes what this PR is about. It should be a clear and concise description concerning what this PR is for, why this PR is needed, and why it should be accepted.-->
<!--Linking an issue can be used alternatively to writing a description.-->

## Implementation Details
<!--Any implementations in this PR that should be carefully looked over, or that could/should have alternate solutions proposed.-->

## Outcome
<!--A short description of what this PR added/fixed/changed/removed.-->
<!--For correct linking of issues please use any of the Closes/Fixes/Resolves keywords. Example: When a PR is fixing a bug use "Fixes: #number-of-bug"-->

## Additional Information
<!--This section is for screenshots to demonstrate any GUI or rendering changes, or any other additional information that reviewers should be aware of.-->

## Potential Compatibility Issues
<!--This section is for defining possible compatibility issues. It must be used when there item/block/material/machine changes, or recipe changes.-->

<!--**Please fill in as much useful information as possible. Also, please remove all unused sections, including this and the other explanations.**-->

clarkins1 committed:
Add files via upload

clarkins1 committed:
Update ru_RU.lang

Russian language update supersymmetry core

clarkins1 committed:
Update ru_RU.lang

clarkins1 committed:
Merge pull request #2 from clarkins1/clarkins1-patch-2

Update ru_RU.lang

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -44,6 +44,7 @@ susy.material.meta_xylene=м-ксилол
 susy.material.para_xylene=п-ксилол
 susy.material.meta_para_xylene_mixture=Смесь м/п-ксилола
 susy.material.phosgene=Фосген
+susy.material.dichloroethane=Дихлорэтан
```
Зачем? Его заменили на соединения чуть пониже

clarkins1 replied:
он был в eu версии файла lang, вот я и вписал

trexxet replied:
Уже нет https://github.com/SymmetricDevs/Supersymmetry/blob/44834ea4a81d901fadb8db8f00726e7a966e0f2d/resources/langfiles/lang/en_us.lang#L47

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -375,10 +376,10 @@ susy.material.rich_amine=Насыщенный амин
-susy.material.furfural=Фурфурал
-susy.material.btex_extract=БТЭК извлекатель
-susy.material.btex=БТЭК
+susy.material.naphtha_reformate=Нафта прошедшая реформинг
```
Реформированный лигроин
По-русски вообще следует везде нафту на лигроин поменять

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -375,10 +376,10 @@ susy.material.rich_amine=Насыщенный амин
-susy.material.btex=БТЭК
+susy.material.naphtha_reformate=Нафта прошедшая реформинг
+susy.material.furfural=Фурфурол
+susy.material.btex_extract=Эксткакт Б.Т.Э.К. 
```
Можно оставить BTEX

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -574,7 +575,7 @@ susy.material.wacker_catalyst=Катализатор Wacker
 susy.material.sodium_sulfate_solution=Раствор сульфата натрия
 susy.material.impregnated_copper_oxide_catalyst=Катализатор на основе пропитанного оксида меди
-susy.material.philips_catalyst=Филлипс Катализатор
+susy.material.philips_catalyst=Катализатор Филлипса 
```
Катализатор Philips, это торговая марка (с одной L!!)

clarkins1 replied:
не понял тебя, ты говоришь что "Катализатор Филлипса" неправильно ?

trexxet replied:
а не, ок - гуглил какие наименования чаще используются и ошибся

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -1563,15 +1571,15 @@ susy.material.roasted_pyrochlore=Пирохлор жареный
 susy.material.impure_niobium_pentachloride=Нечистый пентахлорид ниобия
 susy.material.impure_fluoroniobic_acid_solution=Неочищенный раствор фторниобиевой кислоты
-susy.material.niobium_rich_extract=Экстракт, богатый ниобием
+susy.material.niobium_rich_extract=Экстракт богатый ниобием
```
Обогащённый экстракт ниобия
(ниже аналогично)

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -375,10 +376,10 @@ susy.material.rich_amine=Насыщенный амин
+susy.material.naphtha_reformate=Нафта прошедшая реформинг
+susy.material.furfural=Фурфурол
+susy.material.btex_extract=Эксткакт Б.Т.Э.К. 
+susy.material.btex=Б.Т.Э.К.
```
Аналогично

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -1563,15 +1571,15 @@ susy.material.roasted_pyrochlore=Пирохлор жареный
 susy.material.impure_fluorotantalic_acid_solution=Неочищенный раствор фтортанталовой кислоты
 susy.material.tantalum_rich_extract=Экстракт богатый танталом
-susy.material.scrubbed_tantalum_rich_extract=Очищенный экстракт тантала
+susy.material.scrubbed_tantalum_rich_extract=Очищенный экстракт богатый танталом
```
Ощиченный обогащённый экстракт тантала

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -1751,7 +1759,7 @@ susy.material.supreme_diesel=Высший Дизель
 susy.material.premium_kerosene=Керосин премиум-класса
 susy.material.supreme_kerosene=Высший керосин
-susy.material.crude_lubricating_oil=Сырая смазочная нефть
+susy.material.crude_lubricating_oil=Сырая смазка
```
смазочное масло

clarkins1 committed:
Update ru_RU.lang

trexxet commented on resources/langfiles/lang/ru_RU.lang:
```diff
@@ -1759,7 +1759,7 @@ susy.material.supreme_diesel=Высший Дизель
 susy.material.premium_kerosene=Керосин премиум-класса
 susy.material.supreme_kerosene=Высший керосин
-susy.material.crude_lubricating_oil=Сырая смазка
+susy.material.crude_lubricating_oil=Смазочное масло
```
Crude Lubricating Oil - Сырое смазочное масло
Lubricating Oil - Смазочное масло
Нам надо как-то различать lubrincant и lubricating oil

clarkins1 committed:
Update ru_RU.lang

clarkins1 commented:
еще нефть подправил

clarkins1 commented:
так, отбой, не то загрузил

clarkins1 committed:
Update ru_RU.lang

clarkins1 commented:
никак от этого дихлорбензола избавиться не могу

clarkins1 committed:
Update ru_RU.lang

clarkins1 commented:
сори мужик, только осваиваю гитхаб, в пальцах путаюсь

trexxet commented:
Перевод названий - ок, буду смотреть квесты

Скоро будет обновление, там много чего переделают (электрические схемы и нефтепереработка)

clarkins1 commented:
я доступ открыл, в случае чего, или здесь этого не стоит делать ?

trexxet commented:
> я доступ открыл, в случае чего, или здесь этого не стоит делать ?

в смысле?  Maintainers are allowed to edit this pull request - это?

clarkins1 commented:
Нет, вот это "Allow edits by maintainers"

clarkins1 committed:
Update ru_RU.lang

translation has been updated to version 1.13.5 as of 12/23/2024
перевод подтянут до версии 1.13.5 по состоянию на 23.12.2024

clarkins1 committed:
Update ru_RU.lang

clarkins1 committed:
Update ru_RU.lang

clarkins1 committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

clarkins1 committed:
Update ru_RU.lang

clarkins1 commented:
обновлено до версии 0.1.14.0

clarkins1 committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

clarkins1 committed:
Update SilverChain.groovy
